assets/css/sensei-course-theme/course-settings-editor.scss

```diff
@@ -0,0 +1,27 @@
+.sensei-course-theme-toggle {
+	margin-bottom: 20px;
```

Comment:
I noticed we don't have the "sensei-course-theme-toggle" class applied to any place. Maybe after a refactor and we forgot to remove it?
I think we could clear these styles removing this margin, and the `.components-base-control__field`, or fix it if needed.

We probably could also remove the `&__label` as mentioned here: https://github.com/Automattic/sensei/pull/4565#discussion_r781484490

Reply:
Removed the styles. 👍